docs/jazzy.yml

```diff
@@ -78,7 +78,6 @@ custom_categories:
       - StyleManagerDelegate
       - DayStyle
       - NightStyle
-      - StyleType
```

Comment:
Thanks, my mistake. At the same time we'll eventually have to write documentation for these docs. I'll create a follow-up ticket.